src/gui/screen_menu_move.cpp

```diff
@@ -147,6 +149,9 @@ bool ScreenMenuMove::IsTempOk() {
 
 ScreenMenuMove::ScreenMenuMove()
     : ScreenMenuMove__(_(label)) {
+#if PRINTER_TYPE != PRINTER_PRUSA_MINI
+    header.SetIcon(&png::move_16x16);
+#endif
```

Comment:
Similar as above.

Reply:
MK4 has this icon, MINI das not
it does not change anything, just makes the same code public/private